lib/config-list.ts

```diff
@@ -31,6 +30,7 @@ function configToDescription(config: Config): string | undefined {
 }
 
 function generateConfigListMarkdown(
+  context: Context,
```

Comment:
I always put `context` as the first parameter of a function when it was needed, for consistency.